ui.af.apps/src/main/content/jcr_root/apps/core/fd/af-clientlibs/core-forms-components-runtime-all/resources/i18n/en.json

```diff
@@ -4,6 +4,7 @@
   "FileNameInvalid"             :   "Do not attach files where filename starts with (.), contains \\ / : * ? \" < > | ; % $, or is a reserved keyword like nul, prn, con, lpt, or com.",
   "FileMimeTypeInvalid"         :   "File(s) ${0} are unsupported file types",
   "InternalFormSubmissionError" :   "Encountered an internal error while submitting the form.",
+  "FileSizeZero"                :   "The uploaded file(s) ${0} are empty. Make sure you're uploading file(s) with content.",
```

Comment:
Could you update the other locales with their respective translations? This one is only for English.